pkg/kubelet/container/runtime_cache.go

```diff
@@ -94,3 +94,25 @@ func (r *runtimeCache) getPodsWithTimestamp() ([]*Pod, time.Time, error) {
 	pods, err := r.getter.GetPods(false)
 	return pods, timestamp, err
 }
+
+// TestRunTimeCache embeds runtimeCache with some additional methods for testing.
```

Comment:
That approach was vetoed for Kubernetes. This sounds like more of a Godeps issue to me, but I'm ok with it for now since this file doesn't have any dependencies.

However, could you put it in a separate file from runtime cache (e.g. `test_runtime_cache` or `fake_runtime_cache`), and add a TODO to refactor it to a test-only package?